homeassistant/components/swiss_public_transport/helper.py

```diff
@@ -0,0 +1,15 @@
+"""Helper functions for swiss_public_transport."""
+
+from types import MappingProxyType
+from typing import Any
+
+from .const import CONF_DESTINATION, CONF_START, CONF_VIA
+
+
+def unique_id_from_config(config: MappingProxyType[str, Any] | dict[str, Any]) -> str:
```

Comment:
Move this to `config_flow.py` as it's only used there

Reply:
No, it is also used in the __init__.py file.